apps/test-app/src/generated-types.ts

```diff
@@ -1,11 +1,24 @@
-import { GraphQLResolveInfo } from 'graphql';
```

Comment:
Might want to ignore this file too. It's generated

Reply:
Yeah I just rolled back that commit. I'm not going to backfill prettier on already committed files since the lint-staged task will fix them up if they ever get edited again.